index.html

```diff
@@ -5,8 +5,157 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Africa's talking</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="css/style.css">
+
 </head>
 <body>
+    <nav class="navbar">
+        <div class="navbar-mobile">
```

Comment:
- [ ] [OPTIONAL] kindly consider reducing the use of `<div>` instead use semantic HTML tags like `section` and `article`, this will help your site with SEO and make your code more readable